src/encoded/static/components/regulome_search.js

```diff
@@ -614,9 +620,9 @@ class RegulomeSearch extends React.Component {
     addNewFiles(searchQuery) {
         return new Promise((ok) => {
             requestSearch(searchQuery).then((results) => {
-                const newFiles = results ? results['@graph'] : [];
+                const newFiles = (results && results['@graph']) ? results['@graph'] : [];
```

Comment:
"@graph" is guaranteed to be there though its length might be zero, so the way you had it before should be fine. The important part was checking `newFiles`’s length instead of its existence.

Reply:
Well, locally I was getting errors without the `&& results['@graph']` because `results['@graph']` would be undefined. This way I see the genome and genes tracks when I run it locally.

Reply:
Oh! Maybe regulome is different. OK.